python-sdk/langwatch/examples/generic_bot_rag_multithreaded.py

```
@@ -0,0 +1,65 @@
+import time
+import uuid
+from dotenv import load_dotenv
+import threading
```

Ruff (0.8.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)